tide_core.install

```diff
@@ -1934,3 +1934,32 @@ function tide_core_update_8091() {
     }
   }
 }
+
+/**
+ * Enable views_data_export module.
+ */
+function tide_core_update_8092() {
+  if (\Drupal::moduleHandler()->moduleExists('views_data_export') === FALSE) {
+    \Drupal::service('module_installer')->install(['views_data_export']);
+  }
+}
+
+/**
+ * Enable user_created_by_field module.
+ */
+function tide_core_update_8093() {
+  if (\Drupal::moduleHandler()->moduleExists('user_created_by_field') === FALSE) {
+    \Drupal::service('module_installer')->install(['user_created_by_field']);
+  }
+}
+
+/**
+ * Import user list data.
+ */
+function tide_core_update_8094() {
+  $view = 'views.view.user_list_data_export';
+  $config_path = drupal_get_path('module', 'tide_core') . '/config/install';
+  $source = new FileStorage($config_path);
+  $config_storage = \Drupal::service('config.storage');
+  $config_storage->write($view, $source->read($view));
+}
```

Comment:
I think because the code footprint is small here, this is fine, but for future PR's can we use the pattern of moving work into a class and out of the install hooks? Something like https://github.com/dpc-sdp/tide_search/blob/develop/tide_search.install#L113 for example :)

Reply:
Will do. Thanks @krakerag .